libraries/rush-lib/src/logic/pnpm/test/yamlFiles/pnpm-lock-with-catalog.yaml

```diff
@@ -0,0 +1,36 @@
+lockfileVersion: '9.0'
+
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
+
+catalogsChecksum: 1a2b3c4d5e6f7890abcdef1234567890
+
+catalogs:
+  default:
+    react: 18.2.0
+    typescript: 5.3.0
+  frontend:
+    vue: 3.4.0
+
+importers:
+  .:
+    dependencies:
+      react:
+        specifier: 'catalog:'
```

Comment:
Since `catalog:` is just shorthand for `catalog:default`, should we prefer the latter for clarity?